vue.config.js

```diff
@@ -110,6 +110,36 @@ module.exports = {
       .use('svgo-loader')
       .loader('svgo-loader')
       .end();
+
+    /*
+    {
+        test: /@swagger-api\/apidom-.*\.[mc]?js$/,
+        include: /node_modules/,
+        loader: 'babel-loader',
+        options: {
+          plugins: [
+            '@babel/plugin-transform-class-properties',
+            '@babel/plugin-transform-logical-assignment-operators',
+          ],
+          ...defaultJsOptions,
+        },
+      },
+     */
+    config.module
+      .rule('babel-loader-for-apidom')
+      .test(/@swagger-api\/apidom-.*\.[mc]?js$/)
```

Comment:
I think "swagger-client" was used only in sdk@8, is it still needed?

Reply:
Unfortunately many of the older repos (eg. BondingCurve) still require sdk 8 or similar. Therefore having this allows us to still use these dependencies. :cry:

Reply:
Removed :)